lib/npm-check-updates.js

```diff
@@ -157,15 +157,25 @@ function analyzeGlobalPackages(options) {
 
           let instruction = '[package]'
           if (upgraded) {
-            instruction = upgradedPackageNames.map(pkg => pkg + '@' + upgraded[pkg]).join(' ')
+            // Here I think it is more appropriate to change to latest.
```

Comment:
I may not express it clearly.

![image](https://user-images.githubusercontent.com/39861054/89724352-94edfa00-da34-11ea-8e9c-147dbfbb896a.png)

---------------------------
λ ncu -g -p yarn
[====================] 10/10 100%

 cloc  1.0.4  →  2.6.0

ncu itself cannot upgrade global packages. Run the following to upgrade all global packages:

yarn global upgrade cloc@latest

-------------------------------

After running the command, the console will print out the command line code used to update the dependencies.
I hope to be able to copy and run the command line code directly like the output above.
Because sometimes I do not rely on the current plug-in to update the global dependency package.
If I directly copy and run the previously output command, the global dependency package will be directly locked to a certain version.
But from my personal usage habits, this type of globally installed package usually belongs to the tool or scaffolding category.
So I think this change is reasonable.
I cannot confirm whether this conflicts with your design.
If there is a conflict, I will restore this code to its original state.

Reply:
> If I directly copy and run the previously output command, the global dependency package will be directly locked to a certain version.

I believe global packages are always installed at a specific version. They are all "directly locked". They do not use version ranges. Therefore `yarn global upgrade cloc@latest` is exactly the same (at time of writing) as `yarn global upgrade cloc@2.6.0`.

Regardless, we **must** print the exact version number since ncu does not always upgrade to the latest, as I originally described, so you'll need to restore this part of the code to its original state.